src/Settings/App.SettingsGroup.json

```diff
@@ -179,20 +172,6 @@
     "default":              false,
     "qgcRebootRequired":    true
 },
-{
-    "name":             "userBrandImageIndoor",
```

Comment:
Both of these are still used in the code. I think the UI for setting them was lost somewhere along the way with the settings ui rework. Create #11780

Reply:
They are used, but not this version of them. The ones in the BrandImageSettings Group are used instead of these

Reply:
Ah, right

Reply:
Actually seems kinda unnecessary to have a new settingsgroup for just these two so maybe they belong in the appsettings. But I suppose it really doesn't matter that much. I think a few of the settings could use a bit of reorganization

Reply:
Not quite sure why/when those moved/duplicated. Was that me? Problem with settings moves is not trouncing existing users stuff.

Reply:
Fair, so is this okay then?

Reply:
Sure, if it's not used. Look at blame and can't quite figure out why the AppSettings variants were added.

Reply:
https://github.com/mavlink/qgroundcontrol/commit/23561c4a5ced26271879bf3e6cf131c6bf241d56